src/app/+home-page/home-page.component.html

```diff
@@ -1,5 +1,6 @@
 <ds-home-news></ds-home-news>
 <div class="container">
+  <p *ngIf="(isAuthenticated | async)">Loggato</p>
```

Comment:
Is this line left here as a test? It breaks the AoT build, as isAuthenticated isn't defined in HomePageComponent

Reply:
yes, I am going to remove it